content/00.build/30.zksync-sso/100.faqs.md

```diff
@@ -0,0 +1,64 @@
+---
+title: FAQs
+description: Frequently asked questions about ZKsync SSO.
+---
+
+## What login options are supported?
+
+Currently, only login with passkeys is supported in ZKsync SSO.
+We are working on adding support for logging in with an Externally Owned Account (EOA) in the future.
+
+## What are passkeys?
+
+Passkeys are digital credentials that eliminate the need for traditional passwords by using asymmetric cryptography for authentication.
+They consist of a key pair:
+
+- **Private Key**: Stored securely on the user's device (e.g., in a secure enclave or Trusted Platform Module) and never leaves the device.
+- **Public Key**: Shared with the relying party (ZKsync SSO Auth Server and associated smart contracts) during registration.
+
+Passkeys are based on the [WebAuthn](https://www.w3.org/TR/webauthn-2/) API and the FIDO2 standards,
+which support various authenticators like built-in device biometrics (fingerprint sensors, facial recognition)
+or external hardware security keys (YubiKey, SoloKeys). This ensures a high level of security and a seamless user experience.
+
+## What recovery options are available if I lose my passkey?
+
+In case you lose your passkeys, ZKsync SSO provides several methods to recover your account:
+
+### Multiple Devices with Passkey Synchronization
+
+You can synchronize your passkeys across multiple devices using services like iCloud Keychain for Apple devices or similar services on other platforms.
+This means if one device is lost, your passkeys are still accessible on other synchronized devices.
+
+### Adding Another Passkey (*coming soon*)
+
+You will be able to register a new device by adding a new passkey to your account. This typically involves verifying your identity on the new device,
+after which the new passkey is securely associated with your account.
+
+### Recovery via Externally Owned Account (EOA) (*coming soon*)
+
+You will soon be able to link an Externally Owned Account (EOA) to your ZKsync SSO account. If passkeys are lost,
+you can recover access by authenticating with your EOA, providing an additional layer of security and recovery options.
+
+### Recovery via OAuth (*coming soon*)
+
+The OAuth Recovery ERC-7579 module enables you to set up account recovery through OAuth tokens associated with your existing Web2 logins like
+Gmail, Facebook, or X (formerly Twitter). This allows you to regain access by logging in with your familiar social media or email accounts.
+
+## How secure is ZKsync SSO?
+
+ZKsync SSO uses advanced cryptographic techniques and secure authentication methods like passkeys to ensure that your account remains secure.
```

Comment:
This feels very market-y, can we be more specific what we mean by "advanced cryptographic techniques"?

Reply:
good point, removed